charts/akeyless-api-gateway/templates/_helpers.tpl

```diff
@@ -125,6 +125,34 @@ Generate chart secret name
         {{- end -}}
 {{- end -}}
 
+{{/* Define REDIS_MAXMEMORY as 80% of the pod's memory limit */}}
+{{- define "akeyless-api-gw.redisMaxmemory" -}}
+{{- $memoryLimit := .Values.cache.resources.limits.memory -}}
+
+{{- /* Convert to string to ensure proper handling */}}
+{{- $memoryLimitStr := $memoryLimit | toString -}}
```

Comment:
Why do you need `memoryLimit` and `memoryLimitStr` drop one and use the `toString` function directly

Reply:
Fixed.